src/common/storage/src/operator.rs

```diff
@@ -108,7 +112,8 @@ pub fn build_operator<B: Builder>(builder: B) -> Result<Operator> {
         .layer(LoggingLayer::default())
         // Add tracing
         .layer(MinitraceLayer)
-        // TODO(liyz): add PrometheusClientLayer
+        // Add PrometheusClientLayer
+        .layer(PrometheusClientLayer::new(&mut registry))
```

Comment:
how about move the registry inside a code block here? 

```
.layer({
  let mut registry = load_global_prometheus_registry();
  PrometheusClientLayer::new(&mut registry);
})
```

the `registry` is actually a mutex guard, thus we can reduce the scope of the mutex using the code block.

Reply:
Nice!